app/components/meetings/sessions/edit-intervention-modal.vue

```diff
@@ -213,6 +234,16 @@ function onOrganizationChange(o) {
     this.title              = `${o.name} ${(o.acronym||'') && `(${o.acronym})`}`;
 }
 
+function isUserKronos(id) {
+    const pattern = /^[a-fA-F0-9]{24}$/;
+
+    if(pattern.test(id)) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
`pattern.test()` already return a boolean value. the `if` condition is useless
```suggestion
    return pattern.test(id);
```